shared/settings/nav/index.native.js

```diff
@@ -103,7 +103,7 @@ function SettingsNav({badgeNotifications, badgeNumbers, selectedTab, onTabChange
                     badgeNumber: 0,
                     icon: 'iconfont-nav-settings',
                     onClick: () => onTabChange(Constants.devMenuTab),
-                    text: 'Dev Menu',
+                    text: 'Dev menu',
```

Comment:
Same goes here. Titles should be capitalized, but it won't make a difference because we use all caps for these menu items.